kythe/cxx/indexer/textproto/analyzer.cc

```diff
@@ -68,13 +68,13 @@ constexpr int kNonRepeatedFieldIndex = -1;
 class LoggingMultiFileErrorCollector
     : public google::protobuf::compiler::MultiFileErrorCollector {
  public:
-  void AddError(const std::string& filename, int line, int column,
-                const std::string& message) override {
+  void AddError(absl::string_view filename, int line, int column,
+                absl::string_view message) override {
     LOG(ERROR) << filename << "@" << line << ":" << column << ": " << message;
   }
 
-  void AddWarning(const std::string& filename, int line, int column,
-                  const std::string& message) override {
+  void AddWarning(absl::string_view filename, int line, int column,
```

Comment:
Why by value here vs. by reference above?

(I don't know what's correct, I just noticed they're different, although both are replacing `const std::string&`).

Reply:
By value. std::string_view is a lightweight "reference type" intended to be passed by value.

Reply:
@creachadair thanks, that was a typo. Fixed